src/javaRosa/itextWidget.js

```diff
@@ -305,6 +305,7 @@ define([
                 if (e.form === widget.form && e.itextType === widget.itextType) {
                     if (widget.getItextValue() === e.prevValue) {
                         // Make sure all the defaults keep in sync.
+                        // why doesn't setValue set the itext value?
```

Comment:
that does raise an error here: https://github.com/dimagi/Vellum/blob/ecbd0fb30b35cc655e834408034437e4ff1a40f6/tests/javaRosa.js#L318

I'd have to look into it more deeply to understand why that happens, but thinking it's probably out of scope for this PR